src/pages/library/dataBrowser-utils.test.js

```diff
@@ -0,0 +1,29 @@
+import '@testing-library/jest-dom'
+
+import { brands } from 'src/libs/brands'
+import { dataCatalogStore } from 'src/libs/state'
+import { datasetAccessTypes, filterAndNormalizeDatasets } from 'src/pages/library/dataBrowser-utils'
+
+
+beforeEach(() => {
+  dataCatalogStore.reset()
+})
+
+describe('dataBrowser-utils', () => {
+  it('sets external datasets to accessLevel external', () => {
+    const normalizedDatasets = filterAndNormalizeDatasets(
+      [{ 'dcat:accessURL': 'any-url.com' }],
+      brands.terra.catalogDataCollectionsToInclude
+    )
+    expect(normalizedDatasets[0].access).toBe(datasetAccessTypes.EXTERNAL)
+  })
+
+  it('doesn\'t set non external datasets to accessLevel external', () => {
+    const normalizedDatasets = filterAndNormalizeDatasets(
+      [{ 'dcat:accessURL': 'any-url.com/#workspaces/a/b' }, { 'dcat:accessURL': 'any-url.com//snapshots/details/' }],
```

Comment:
This is the kind of test that I doubt the value of. This is (mostly) testing if a string literal in the code is the same as a string literal in the test. If the URLs in the data change, our code would be wrong but the test would still pass. If the URLs in the data change and the code is updated, our tests would fail until we fix them, but that failure didn't tell us anything interesting about the code being tested.

Changing the test and code to use a shared constant would be an improvement as you'd no longer have a test failure due to updating a string in the code being tested.